components/MovieCard/index.js

```diff
@@ -0,0 +1,15 @@
+import Image from "next/image";
+
+const MovieCard = ({ title, director, image }) => {
+  return (
+    <div className="movie-card">
+      <Image src={image} alt="Movie Poster" width={225} height={330} />
+      <aside className="movie-card__info">
```

Comment:
you are required to use styled components in this project :/ so please only use that styling approach and no classes.